src/main/java/org/opensearch/security/dlic/rest/api/ssl/CertificateType.java

```diff
@@ -1,48 +0,0 @@
-/*
- * SPDX-License-Identifier: Apache-2.0
- *
- * The OpenSearch Contributors require contributions made to
- * this file be licensed under the Apache-2.0 license or a
- * compatible open source license.
- *
- * Modifications Copyright OpenSearch Contributors. See
- * GitHub history for details.
- */
-
-package org.opensearch.security.dlic.rest.api.ssl;
-
-import java.util.Locale;
-
-public enum CertificateType {
```

Comment:
Why remove this enum?

Reply:
The main issue is the use of duplicate definitions for certificate types, which prevents the REST endpoint from providing information about transport client certificates. These certificates can be configured differently from standard transport certificates under an extended SSL configuration.